OpenRA.Game/Game.cs

```diff
@@ -172,6 +172,38 @@ internal static void StartGame(string mapUID, WorldType type)
 			GC.Collect();
 		}
 
+		public static void RestartGame()
```

Comment:
Maybe move https://github.com/OpenRA/OpenRA/blob/bleed/OpenRA.Mods.Common/Widgets/Logic/MissionBrowserLogic.cs#L311-L326 into this file, add a map uid parameter to it, rename it to something more appropriate, and call it from MissionBrowserLogic.